core-bundle/contao/drivers/DC_Table.php

```diff
@@ -6643,4 +6641,28 @@ private function addDynamicPtable(array $data): array
 
 		return $data;
 	}
+
+	private function isGrantedToClipboard(array $arrClipboard, array $data): bool
```

Comment:
Is that only in DC_Table?

Reply:
yes, neither the DC_Folder nor DC_File have a `paste_button_callback` or anything thelike. There are currently no permissions on move operations in DC_Folder, and you obviously can't move something in DC_File.